src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml

```diff
@@ -69,9 +69,14 @@ sylius_fixtures:
                     options:
                         random: 3
 
-                user:
+                regular_user:
+                    name: user
```

Comment:
I come up with the following naming:
 - `regular_user` is an alias
 - `user` is a fixture name

It is based on `FixtureInterface::getName()` method (and similar to `ListenerInterface::getName()`).

I'm not sure for `type` too, as IMO it assumes that there is some closed list of available choices.